week-2-practise/level-2/5-indexOf.js

```diff
@@ -5,7 +5,9 @@
 // TIP: Use the .indexOf() method
 
 function remove(arr, valueToRemove) {
-  return; // complete this statement
+  var valueToRemove = arr.indexOf();
+  var 
+  return ; // complete this statement
 }
 
 /* 
```

Comment:
Write a function that removes an element from an array 
Tips: use  `.slice() and .indexOf()`